.changelog/2100.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:enhancement
```

Comment:
it's ok although i thought you'll use a PR per file

Reply:
noted in PR description, this is an exception as I am covering several PRs to be "up to date", moving forward each PR will handle its changelog entry if needed.